cypress/integration/route_spec.js

```diff
@@ -80,13 +81,93 @@ describe('Sea route tests', function () {
     });
 
     it('Save template', function () {
-        cy.contains('Tallenna luonnoksena').click();
-        cy.get('tr').should('have.length', 2);
+        cy.contains('Tallenna ja julkaise').click();
+        cy.contains('Reitin tallennus onnistui');
     });
 
-    it('Delete template', function () {
-        cy.get('[id*="delete-route"]').click();
+    it('Link route to service - create new service', function () {
+        const service = this.service;
+        cy.visit('/#/own-services');
+
+        // Add new schedule type service
+        cy.server();
+        cy.route('POST', '/transport-service').as('addService');
+
+
+        cy.get('a[id*="new-service-button"]').click({force: true});
+
+        cy.get('[id*="Valitseliikkumispalveluntyyppi"]').click();
+
+        cy.server();
+        cy.route('/place-completions/*').as('placeCompletion');
+
+        cy.contains(/^Säännöllinen*/).click();
```

Comment:
Would it be better to get an element and then check does that contain the value?

Reply:
Maybe. In cypress documents they suggest to use .contains() as follows:
cy.get('form').contains('submit the form!').click()
but I think that it would make a big difference.